### PR TITLE
Quick adjust to freight reader

### PR DESCRIPTION
**Pull request by haitamlaarabi**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3455)

### CODE REVIEW

**Comment by haitamlaarabi** [comment]
Test now!

File: src/main/scala/beam/agentsim/agents/freight/input/GenericFreightReader.scala [hunk 1]
```diff
@@ -122,41 +201,86 @@ class GenericFreightReader(val config: Freight, val geoUtils: GeoUtils, rnd: Ran
...
+
+  private def getTaz(tazId: String): TAZ = tazTree.getTAZ(tazId) match {
+    case Some(taz) => taz
+    case None      => throw new IllegalArgumentException(s"Cannot find taz with id $tazId")
+  }
+
+  private def getDistributedTazLocation(taz: TAZ): Coord =
+    convertedLocation(TAZTreeMap.randomLocationInTAZ(taz, rnd))
+
+  private def checkNullOrEmpty(str: String): Boolean = str == null || str.isEmpty
```
Comment (dimaopen) [inline src/main/scala/beam/agentsim/agents/freight/input/GenericFreightReader.scala]: We could use `import org.apache.commons.lang3.StringUtils.isBlank` instead.

**Review by dimaopen — APPROVED** [review]
LGTM, except not using `isBlank` of lang3 library. There's a compile error during merging I think.

**Comment by haitamlaarabi** [comment]
Test now!

**Comment by haitamlaarabi** [comment]
Test now!

**Pull request merged**